boards/adafruit-qtpy-esp32-pico/definition.json

```diff
@@ -0,0 +1,101 @@
+{
+    "boardName":"adafruit-qtpy-esp32-pico",
+    "mcuName":"esp32",
+    "mcuRefVoltage":3.3,
+    "displayName":"Adafruit QT Py ESP32 Pico",
```

Comment:
can you remove the Pico from the end? its the name of the chip but probably not the name of the board

Reply:
Removed `pico` from both the definition and the folder name, https://github.com/adafruit/Wippersnapper_Boards/pull/38/commits/05c9af26141617b3e22eeedb6f5e3f366454cd39